### PR TITLE
Add prevent text selection while resizing

### LINKED ISSUE #33: Prevent text selection while resizing
[Issuehunt badges](https://issuehunt.io/r/egoist/demoboard/issues/33)

Kapture 2019-08-04 at 14 05 57

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Backers (Total: $50.00)

- [issuehunt issuehunt](https://issuehunt.io/u/issuehunt) ($50.00)
### Submitted pull Requests
- [#34 Add prevent text selection while resizing](https://issuehunt.io/r/egoist/demoboard/pull/34)
---

### Tips

- Checkout the [Issuehunt explorer](https://issuehunt.io/r/egoist/demoboard/) to discover more funded issues.
- Need some help from other developers? [Add your repositories](https://issuehunt.io/r/new) on IssueHunt to raise funds.
---
IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### PR DESCRIPTION
**Pull request by daltonmenezes**
This PR fixes #33

---

<details>
<summary>
<b>IssueHunt Summary</b>
</summary>

### Referenced issues

This pull request has been submitted to:
- [#33: Prevent text selection while resizing](https://issuehunt.io/repos/195253194/issues/33)
---

IssueHunt has been backed by the following sponsors. [Become a sponsor](https://issuehunt.io/membership/members)
</details>

### CODE REVIEW

File: src/Resizebar.js [hunk 1]
```diff
@@ -14,14 +14,29 @@ export const Resizebar = ({ axis, rootSelector, bounds }) => {
...
+    })
+  }
+
+  const dragend = () => {
+    document.body.style = `
+      -webkit-user-select: text;
+      -moz-user-select: text;
+      -ms-user-select: text;
+    `
   }
```
Comment (egoist) [inline src/Resizebar.js]: What about using the `onselectstart` event?

```js
// drag start
document.onselectstart = () => false

// drag end
document.onselectstart = () => true
```

Comment (daltonmenezes) [reply src/Resizebar.js]: It's really better. Done :)

**Comment by daltonmenezes** [comment]
@egoist Could you please confirm the PR status on issuehunt?

**Pull request merged**